priv/repo/migrations/20160520141230_add_token_to_member.exs

```diff
@@ -0,0 +1,11 @@
+defmodule Community.Repo.Migrations.AddTokenToMember do
+  use Ecto.Migration
+
+  def change do
+    execute "TRUNCATE TABLE members;"
```

Comment:
That makes sense. I have an alternative solution. What if you did not save the token at all and instead generated a token with `Phoenix.Token.sign` and `Phoenix.Token.verify` instead? https://hexdocs.pm/phoenix/1.2.0-rc.0/Phoenix.Token.html

We did this with in Constable: https://github.com/thoughtbot/constable/pull/263/files

That way you don't need the migration, no need to save it to the db. Makes things a bit simpler I think